src/main/resources/upload-your-documents/translation.json

```diff
@@ -25,12 +25,16 @@
     "en": {
     "SIZE_ERROR": "File size exceeds 20Mb. Please upload a file that is less than 20Mb",
     "FORMAT_ERROR": "This service only accepts files in the formats - Ms Word, Ms Excel, PDF, JPG, GIF, PNG, TXT, RTF",
-    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry"
+    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry",
+    "CONTINUE_WITHOUT_UPLOAD_ERROR": "You cannot continue without uploading the application",
+    "NO_FILE_UPLOAD_ERROR": "Please choose a file to upload"
     },
     "cy": {
     "SIZE_ERROR": "File size exceeds 20Mb. Please upload a file that is less than 20Mb - welsh",
     "FORMAT_ERROR": "This service only accepts files in the formats - Ms Word, Ms Excel, PDF, JPG, GIF, PNG, TXT, RTF - welsh",
-    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry - welsh"
+    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry - welsh",
```

Comment:
Please check with Subha and update the story with number of files allowable. I guess the Defect ISDB-280 is already created for that.

Reply:
Already discussed.